src/menu_new_game/A5CC50.c

```diff
@@ -0,0 +1,336 @@
+    "；", "：", "／", "＄", "．", "，", "＜", "＞",
+    "♂", "♀", "［", "］", "ε", "　", "\0",
+};
+
+char* D_800E356C[] = { "←", "→", "　", "End" };
+
+ScreenSettings menu_new_game_video_settings = {
+    D_803B5000, /* fb1 */
+    D_803DA800, /* fb2 */
+    NULL,       /* fb3 */
+    NULL,       /* zbuffer */
+    320,        /* width*/
+    240,        /* height */
+    0x00016A99 /* flags*/
+};
+
+Lights1 D_800E3598 = gdSPDefLights1(100, 100, 100, 180, 180, 180, 30, 30, 30);
+
+Gfx D_800E35B0[] = {
+    gsDPPipeSync(),
+    gsDPSetCycleType(G_CYC_2CYCLE),
+    gsDPSetRenderMode(G_RM_FOG_SHADE_A, G_RM_AA_ZB_OPA_SURF2),
+    gsSPFogFactor(256, -512),
+    gsDPSetFogColor(0, 0, 0, 255),
+    gsSPSetGeometryMode(G_FOG),
+    gsSPSetGeometryMode(G_LIGHTING),
+    gsSPSetLights1(D_800E3598),
+    gsSPEndDisplayList(),
+};
+
+// BSS
+static GObj* D_801180B0;
+static GObj* D_801180B4;
+static GObj* D_801180B8;
+static s32 D_801180C0[20];
+s32 D_80118110;
+s32 D_80118114;
+static u8 D_80118118[0x50000];
+
+void func_800E2ED0_A5E280(void);
+
+void func_800E18A0_A5CC50(SObj* arg0, u8 color) {
+    arg0->sprite.red = color;
+    arg0->sprite.green = color;
+    arg0->sprite.blue = color;
+}
+
+char** func_800E18B4_A5CC64(s32 arg0) {
+    return &D_800E2FF0[255];
+}
+
+void func_800E18C4_A5CC74(void) {
+
+}
+
+void func_800E18CC_A5CC7C(void) {
+    SObj* sobj;
+
+    D_801180B4 = ohCreateSprite(14, ohUpdateDefault, 0, 0x80000000, renDrawSprite, 1, 0x80000000, -1, &D_80109880_A84C30, 0, 0, 1);
+    sobj = D_801180B4->data.sobj;
+    sobj->sprite.attr = SP_TEXSHUF | SP_TRANSPARENT;
+}
+
+void func_800E1950_A5CD00(void) {
+    SObj* sobj;
+
+    D_801180B8 = ohCreateSprite(14, ohUpdateDefault, 0, 0x80000000, renDrawSprite, 1, 0x80000000, -1, &D_80117F98_A93348, 0, 0, 1);
+    sobj = D_801180B8->data.sobj; 
+    sobj->sprite.x = 96; sobj->sprite.y = 21;
+    sobj->sprite.attr = SP_TEXSHUF | SP_TRANSPARENT;
+}
+
+void func_800E19E4_A5CD94(void) {
+    func_800E18CC_A5CC7C();
+    func_800E1950_A5CD00();
+}
+
+void func_800E1A0C_A5CDBC(void) {
+    s32 id;
+    s32 id2;
+    s8 i;
+    ucolor sp28;
+
+    id2 = func_8036F78C_842F3C();
+    func_8036F684_842E34(id2, 1);
+    func_8036F738_842EE8(id2, &sp28);
+    func_8036F1F4_8429A4(id2, 184, 320);
+    func_8036F0DC_84288C(id2, 1);
+    func_8036F0A0_842850(id2, 0);
+    func_8036FE54_843604(id2, 1);
+
+    // description
+    id = func_8036AC6C_83E41C(104, 173, 192, 47, 0);
+    func_8036B870_83F020(id, 1, 255, 255, 255, 255);
+    func_8036B870_83F020(id, 0, sp28.r, sp28.g, sp28.b, 255);
+    func_8036B734_83EEE4(id);
+    func_8036B9EC_83F19C(id, 0, 0);
+    func_8036D448_840BF8(1);
+    func_8036D3E8_840B98(-1, 3);
+    func_8036CB58_840308(id, 12);
+    func_8036C898_840048(id, "Enter your name on the card.\n"
+                             "Choose with the Control Stick\n"
+                             "and select with the \\a Button!");
+
+    // character grid
+    for (i = 0; i < 20; i++) {
```

Comment:
wherever we loop through D_801180C0, it'd be good to use ARRAY_COUNT(D_801180C0) for the end condition instead of the hard-coded 20